src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindOrganisationsByPbaStatusTest.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.professionalapi.controller.advice.ErrorResponse;
+import uk.gov.hmcts.reform.professionalapi.controller.constants.ErrorConstants;
+import uk.gov.hmcts.reform.professionalapi.controller.request.OrganisationCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.response.OrganisationsWithPbaStatusResponse;
+import uk.gov.hmcts.reform.professionalapi.domain.PbaStatus;
+import uk.gov.hmcts.reform.professionalapi.util.AuthorizationEnabledIntegrationTest;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.organisationRequestWithAllFields;
+
+@SuppressWarnings("unchecked")
+public class FindOrganisationsByPbaStatusTest extends AuthorizationEnabledIntegrationTest {
+
+    @Test
+    public void get_request_returns_organisations_with_correct_pba_status() throws JsonProcessingException {
+        createOrganizationsAndEditPbas();
+
+        List<OrganisationsWithPbaStatusResponse> orgPbaResponse = (List<OrganisationsWithPbaStatusResponse>)
+                professionalReferenceDataClient.findOrganisationsByPbaStatus(
+                        PbaStatus.ACCEPTED.toString(), hmctsAdmin, Boolean.FALSE);
+
+        assertThat(orgPbaResponse).isNotNull();
```

Comment:
Not sure if verifying the response not to be null is enough. Would it be possible to check that it is returning all the organisations that are being created in the method - `createOrganizationsAndEditPbas()`?